scripts/installation/install.sh

```diff
@@ -78,7 +78,91 @@ delete_crds() {
   kubectl delete -f build/crds --timeout=90s
 }
 
+create_kb(){
+cd ${SEDNA_ROOT}
+kubectl create namespace sedna
+kubectl label node/$KB_NODE_NAME sedna=kb --overwrite
+
+kubectl $action -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: kb
+  namespace: sedna
+spec:
+  selector:
+    sedna: kb
+  type: NodePort
+  ports:
+    - protocol: TCP
+      port: 9020
+      targetPort: 9020
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kb
+  labels:
+    sedna: kb
+  namespace: sedna
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      sedna: kb
+  template:
+    metadata:
+      labels:
+        sedna: kb
+    spec:
+      nodeSelector:
+        # only schedule to master node
+        node-role.kubernetes.io/master: ""
+      serviceAccountName: sedna
+      containers:
+      - name: kb
+        imagePullPolicy: IfNotPresent
+        image: kubeedge/sedna-kb:v0.1.0
+        command: ["python", "/code/lib/sedna/service/run_kb.py"]
+        env:
+          - name: LIFELONG_KB_URL
+            value: "sqlite:///db/lifelong_kb.sqlite3"
+        volumeMounts:
+        - name: lifelong-kb-url
+          mountPath: /db
+        resources:
+          requests:
+            memory: 256Mi
+            cpu: 512m
+          limits:
+            memory: 512Mi
+      volumes:
+        - name: lifelong-kb-url
+          hostPath:
+            path: /rootfs
```

Comment:
/rootfs should be renamed into something like /opt/kb-data

Reply:
ok,done it